contracts/core/liquidators/twap-impl/TWAPAuction.sol

```diff
@@ -29,8 +41,13 @@ import { TwoAssetPriceBoundedLinearAuction } from "../impl/TwoAssetPriceBoundedL
  * TwoAssetPriceBoundedLinearAuction
  */
 contract TWAPAuction is TwoAssetPriceBoundedLinearAuction {
+    using SafeMath for uint256;
+    using CommonMath for uint256;
+    using AddressArrayUtils for address[];
+    using BoundsLibrary for BoundsLibrary.Bounds;
 
     /* ============ Structs ============ */
+
```

Comment:
Would be good to add javadoc to provide descriptions / definitions of each of the items in the struct